cluster/gce/coreos/helper.sh

```diff
@@ -46,6 +46,11 @@ function create-master-instance() {
   local address_opt=""
   [[ -n ${1:-} ]] && address_opt="--address ${1}"
 
+  local preemptible_master=""
+  if [[ "${PREEMPTIBLE_MASTER}" == "true" ]]; then
```

Comment:
It might be good to be defensive here and use `"${PREEMPTIBLE_MASTER:-}"` so as not to break any case where PREEMPTIBLE_MASTER is not set